test/modules/TestResultCoordinator/TestResultCoordinator.csproj

```diff
@@ -21,16 +21,30 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Logging" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.1.1" />
```

Comment:
make sure use existing version in the solution for all dependencies; e.g. AspNetCore uses 2.2.0.

Reply:
Updated